app/api/dao/mentorship_relation.py

```diff
@@ -162,7 +162,7 @@ def isValidState(rel_state):
 
     @staticmethod
     @email_verification_required
-    def accept_request(user_id: int, request_id: int):
+    def accept_request(user_id: int, relation_id: int):
```

Comment:
No need to do any of the changes(i.e from request_id to relation_id) in this function. Since, we still haven't accepted the relation request.